src/views/Releases/Release/index.jsx

```diff
@@ -152,11 +246,11 @@ export default function Release(props) {
             {!isNewRelease && (
```

Comment:
Similar to the rule view, let's now show the speed dial if the buttons are no changes to cancel.

Reply:
Oh, whoops. I forgot to push before merging, heh.